File: week0/cpp/10872.cpp
#include <iostream>

using namespace std;

int get_factorial(int);

int main() {
    int N;
    cin >> N;
    int factorial = get_factorial(N);
    cout << factorial << endl;
}

int get_factorial(int num) {
    if (num == 0 || num == 1)
        return 1;
    else
        return num * get_factorial(num-1);
}

File: week0/py/10872.py

def run():
    N = int(input())
    factorial = get_factorial(N)
    print(factorial)

def get_factorial(num):
    if num == 0 or num == 1:
        return 1
    else:
        return num * get_factorial(num-1)

if __name__ == '__main__':
    run()